crates/cli/Cargo.toml

```diff
@@ -86,6 +86,11 @@ notify.workspace = true
 
 [dev-dependencies]
 pretty_assertions.workspace = true
+fs_extra.workspace = true
+assert_cmd = "2"
+predicates = "3"
+portpicker = "0.1"
+reqwest = { version = "0.12", features = ["blocking", "json"] }
```

Comment:
I'm not sure if I should add these to the workspace because they're dev only dependencies